qubes_menu/appmenu.py

```diff
@@ -116,6 +117,15 @@ def __init__(self, qapp, dispatcher):
         self.appmenu_position: str = 'mouse'
 
     def _add_cli_options(self):
+        self.add_main_option(
```

Comment:
I don't like this approach, I don't want to expose tests that explicitly to users. Better add a wrapper to run tests or something like that, if you need it outside unit tests.